File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    const arr = s.split(' ').map(Number);
    // const arr = s.split(' ');
    arr.sort((a, b) => a - b);
    return arr[0] + ' ' + arr[arr.length - 1];
    // return Math.min(...arr) + ' ' + Math.max(...arr);
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    array = s.split(' ')
    new_array = []
    for i in range(len(array)):
        array[i] = int(array[i]) 
        
    array.sort()
    answer = str(array[0]) + ' ' + str(array[-1])
    return answer